test/deps/package.json

```diff
@@ -52,7 +52,11 @@
     "jade": "~1.8.1",
     "hamljs": "~0.6.2",
     "handlebars": "~2.0.0",
-    "qrcode-terminal": "~0.9.5"
+    "qrcode-terminal": "~0.9.5",
```

Comment:
@zckrs Proper diff here. I had the same problem in others PR, but fixed them too.